tools/perf_checker/perf_checker.sh

```diff
@@ -0,0 +1,41 @@
+.sh
+#!/bin/sh
+set -euox pipefail
+
+# Installing the requirements
+python -m venv perf
+source perf/bin/activate
+pip install psutil
+pip install gitpython
+pip install somacore
+pip install tiledbsoma
+pip install cellxgene_census
+
+# Installing mount-s3
+sudo wget https://s3.amazonaws.com/mountpoint-s3-release/latest/x86_64/mount-s3.deb
+sudo apt install -y ./mount-s3.deb
+
+# Setting up mount-s3
+mkdir ./mount-s3
+mkdir ./s3_cache
+mount-s3 census-profiler-tests ./mount-s3 --cache ./s3_cache  --metadata-ttl 300
+
+dbpath=`pwd`/mount-s3
+echo "Mount-S path = ${dbpath}"
+
+# New benchmarks must be added to this list
+declare -a benchmarks=("./tools/perf_checker/benchmark1.py")
+
+# Downloading TileDB-SOMA (branch or main once merged)
+git clone https://github.com/single-cell-data/TileDB-SOMA.git
+cd TileDB-SOMA
+git checkout census_profiler
```

Comment:
This will be removed before merging to main, right? (I think that's what the comment above is saying)

Reply:
Correct

Reply:
This was not removed and is now present in `main`, and has broken the CI